src/shader_recompiler/ir/passes/resource_tracking_pass.cpp

```diff
@@ -656,6 +656,32 @@ void PatchImageInstruction(IR::Block& block, IR::Inst& inst, Info& info, Descrip
     }
     ASSERT(image.GetType() != AmdGpu::ImageType::Invalid);
     const bool is_storage = IsImageStorageInstruction(inst);
+
+    // Patch image instruction if image is fmask.
```

Comment:
I would prefer to use `FMask` here and in other places as it is official terminology